poglink/error.py

```diff
@@ -0,0 +1,10 @@
+class RatesWriteError(Exception):
+    pass
+
+
+class RatesFetchError(Exception):
+    pass
+
+
+class RatesProcessError(Exception):
+    pass
```

Comment:
Nothing fancy, just giving them a custom name so they're more specific to what actually went wrong.